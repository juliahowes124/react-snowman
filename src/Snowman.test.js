import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Snowman from './Snowman';

describe('proper rendering', () => {
  it('renders without crashing', () => {
    render(<Snowman/>);
  })
  //test vs it?
  it('matches snapshot at endgame', () => {
    const { container } = render(<Snowman/>);

    // container.querySelector('button[value="b"]').fireEvent('click')
    // container.querySelector('button[value="c"]').fireEvent('click')
    // container.querySelector('button[value="d"]').fireEvent('click')
    // container.querySelector('button[value="f"]').fireEvent('click')
    // container.querySelector('button[value="g"]').fireEvent('click')
    // container.querySelector('button[value="h"]').fireEvent('click')

    // expect(container).toMatchSnapshot();
  })
})