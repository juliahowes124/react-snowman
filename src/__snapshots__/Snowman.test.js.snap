// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proper rendering matches snapshot at endgame 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number Wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p>
      You Lose! The correct answer was 
      <strong>
        apple
      </strong>
    </p>
  </div>
</div>
`;
